### PR TITLE
Update notification drawer to use trackByID

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Depends on release including [PR 180](https://github.com/openshift/origin-web-common/pull/180) from origin-web-common.

@spadgett @jeff-phillips-18

### CODE REVIEW

File: app/views/directives/notifications/notification-drawer-wrapper.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   heading-include="{{$ctrl.headingInclude}}"
   notification-body-include="{{$ctrl.notificationBodyInclude}}"
   notification-groups="$ctrl.notificationGroups"
-  notification-track-field="uid"
+  notification-track-field="trackByID"
```
Comment (spadgett) [inline app/views/directives/notifications/notification-drawer-wrapper.html]: Don't you need to set `trackByID` here for events?

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/notifications/notificationDrawerWrapper.js#L161

Comment (benjaminapetersen) [reply app/views/directives/notifications/notification-drawer-wrapper.html]: Yup! looking at that now, I'll have an update in a second.

Comment (benjaminapetersen) [reply app/views/directives/notifications/notification-drawer-wrapper.html]: I think mapping `event.metadata.uid` is fine.  not sure if it really needs the same format of ` _.uniqueId('notification-') + Date.now()`

Comment (benjaminapetersen) [reply app/views/directives/notifications/notification-drawer-wrapper.html]: yeah has to be this way with `count` increments & things.

**Comment by benjaminapetersen** [comment]
updated

**Comment by spadgett** [comment]
[merge][severity: bug]

**Comment by spadgett** [comment]
Flake #1685 (not the one we just fixed!)

[merge][severity: bug]

**Pull request merged**